packages/manager/apps/catalog-revamp/src/components/Filters/Filters.scss

```diff
@@ -0,0 +1,21 @@
+.filters-container {
+  text-align: left;
+
+  @media (max-width: 639px) {
```

Comment:
use ods design tokens

Reply:
Done need to use --ods-blue-jeans-breakpoint-100 you are right